e2e-tests/insurance/cypress/e2e/journeys/policy/other-company-details/changing-another-company-from-yes-to-no.spec.js

```diff
@@ -0,0 +1,76 @@
+import { field as fieldSelector } from '../../../../../../pages/shared';
+import { FIELD_VALUES } from '../../../../../../constants';
+import { POLICY as POLICY_FIELD_IDS } from '../../../../../../constants/field-ids/insurance/policy';
+import { INSURANCE_ROUTES } from '../../../../../../constants/routes/insurance';
+
+const {
+  ROOT,
+  POLICY: { OTHER_COMPANY_DETAILS },
+} = INSURANCE_ROUTES;
+
+const {
+  REQUESTED_JOINTLY_INSURED_PARTY: {
+    REQUESTED, COMPANY_NAME, COMPANY_NUMBER, COUNTRY,
+  },
+} = POLICY_FIELD_IDS;
+
+const baseUrl = Cypress.config('baseUrl');
+
+context(`Insurance - Policy - Other company details page - Changing ${REQUESTED} from 'yes' to 'no'`, () => {
+  let referenceNumber;
+  let url;
+
+  before(() => {
+    cy.completeSignInAndGoToApplication({}).then(({ referenceNumber: refNumber }) => {
+      referenceNumber = refNumber;
+
+      // go to the page we want to test.
+      cy.startInsurancePolicySection({});
+
+      cy.completeAndSubmitPolicyTypeForm(FIELD_VALUES.POLICY_TYPE.SINGLE);
+      cy.completeAndSubmitSingleContractPolicyForm({});
+      cy.completeAndSubmitTotalContractValueForm({});
+      cy.completeAndSubmitNameOnPolicyForm({ sameName: true });
+      cy.completeAndSubmitPreCreditPeriodForm({});
+      cy.completeAndSubmitAnotherCompanyForm({ otherCompanyInvolved: true });
+
+      url = `${baseUrl}${ROOT}/${referenceNumber}${OTHER_COMPANY_DETAILS}`;
+
+      cy.assertUrl(url);
+    });
+  });
+
+  beforeEach(() => {
+    cy.saveSession();
+  });
+
+  after(() => {
+    cy.deleteApplication(referenceNumber);
+  });
+
+  describe(`when changing ${REQUESTED} from 'yes' to 'no'`, () => {
+    beforeEach(() => {
+      cy.navigateToUrl(url);
+
+      cy.completeAndSubmitOtherCompanyDetailsForm();
+
+      /**
+       * Go back to the "other company"
+       * So we can change "yes" to no"
+       */
+      cy.go('back');
```

Comment:
could we not `navigateToUrl `instead?

Reply:
I was thinking, "going back" is representative of the user journey whereas `navigateToUrl` is not (manually changing the URL). Although this is quicker for E2E tests. Can I update in the next PR?

Reply:
next PR